parseConformance.js

```diff
@@ -347,6 +347,9 @@ class ParseConformance {
                         _required: backboneElement.min === 1,
                         _properties: []
                     };
+                    if (parentBackboneElement._properties === undefined || parentBackboneElement._properties === null) {
```

Comment:
Could probably be simplified to `!parentBackboneElement`, but will accept as-is... mainly a coding preference thing

Reply:
Or even `parentBackboneElement._properties = parentBackboneElement._properties || [];`